.github/workflows/live-test.yml

```diff
@@ -0,0 +1,57 @@
+name: Live tests for B2K
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+jobs:
+  liveTest:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    name: live test for b2k
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup kubectl for macos-latest
+      if: ${{ matrix.os == 'macos-latest' }}
+      uses: azure/setup-kubectl@v3
+      with:
+       version: latest
+    - name: Start minikube
+      uses: medyagh/setup-minikube@master
```

Comment:
we should pin this to a sha instead of master. Master is a big security vulnerability.

Reply:
done updated to sha. good suggestion, thank you.